modules/end-user-portal/src/Core/ContainerModule.cs

```diff
@@ -0,0 +1,13 @@
+using Autofac;
+using GpConnect.NationalDataSharingPortal.EndUserPortal.Core.Data;
+using GpConnect.NationalDataSharingPortal.EndUserPortal.Core.Data.Interfaces;
+
+namespace GpConnect.NationalDataSharingPortal.EndUserPortal.Core;
+
+public class ContainerModule : Module
```

Comment:
Do we need to add AutoFac just for this, 

we can do serviceCollection.AddScoped<Interface:Class>();